### PR TITLE
Address style nits from bitemyapp/bloodhound#266

### PR DESCRIPTION
**Pull request by ivanbakel**
I'm really grateful that someone has taken the effort to bring ES7 support to Bloodhound. To support those efforts, I've taken the liberty of fixing the menial style nits raised in the Bloodhound PR.

@AlexeyRaga

### CODE REVIEW

**Comment by joshgodsiff** [comment]
Are you sure you didn't mean to raise this PR against the `bitemyapp` main branch, not the `packetloop` fork?

**Comment by ivanbakel** [comment]
I'm trying to contribute to the changeset for the PR bitemyapp/bloodhound#266. I'm doing this to correct style nits that were raised in that PR about the changes present on the `packetloop` fork. These corrections can't be made on the `bitemyapp` fork, because the changes have not been merged in.

**Comment by joshgodsiff** [comment]
Ah, I see!

Unfortunately, most of the team that was maintaining this fork got made redundant. I don't think Alexey or I have permissions to merge it any longer.

I'd suggest submitting your own PR with both our changes and your fixes directly to `bitemyapp`.

**Comment by joshgodsiff** [comment]
You could also try getting in contact with @tttrinh or @rahmed18 - who I believe still have access (or can point you at someone who does).

File: src/Database/Bloodhound/Types.hs [hunk 1]
```diff
@@ -619,14 +619,12 @@ instance FromJSON TimeUnits where
...
-data SearchTemplate =
-  SearchTemplate {
-      searchTemplate                      :: Either SearchTemplateId SearchTemplateSource,
-      params                              :: TemplateQueryKeyValuePairs,
-      explainSearchTemplate               :: Maybe Bool,
-      profileSearchTemplate               :: Maybe Bool
-    }
-  deriving (Eq, Show)
+data SearchTemplate = SearchTemplate {
+    searchTemplate                      :: Either SearchTemplateId SearchTemplateSource
```
Comment (AlexeyRaga) [inline src/Database/Bloodhound/Types.hs:623]: Since it is a styling issue, shouldn't `::` be aligned here?

**Comment by AlexeyRaga** [comment]
I don't have permissions anymore, but 👍  from me if it counts :)

**Comment by andrewbeard** [comment]
> I don't have permissions anymore, but 👍 from me if it counts :)

If Alexey says it's good then it sounds good to me!

**Pull request merged**